portal-api/api-tests/v1/utilisation-report-service/utilisation-report-upload.api-test.js

```diff
@@ -18,7 +18,7 @@ jest.mock('../../../src/drivers/fileshare', () => ({
 }));
 
 jest.mock('../../../src/v1/api', () => ({
-  saveUtilisationReport: jest.fn().mockResolvedValue({ status: 201, data: { dateUploaded: new Date() } }),
+  saveUtilisationReport: jest.fn().mockResolvedValue({ dateUploaded: new Date() }),
```

Comment:
This is where the api tests error comes from - we check that the length of `response.data` is equal to 1, but we mock a single object return here. Need to add some docs to make it more clear as to what is returned